install

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Reset
+Color_Off='\033[0m' # Text Reset
+
+# Regular Colors
+Red='\033[0;31m'   # Red
+Green='\033[0;32m' # Green
+Dim='\033[0;2m'    # White
+
+# Bold
+Bold_Green='\033[1;32m' # Bold Green
+Bold_White='\033[1m'    # Bold White
+
+error() {
+    echo -e "${Red}error${Color_Off}:" "$@" >&2
+    exit 1
+}
+
+info() {
+    echo -e "${Dim}$@ ${Color_Off}"
+}
+
+info_bold() {
+    echo -e "${Bold_White}$@ ${Color_Off}"
+}
+
+success() {
+    echo -e "${Green}$@ ${Color_Off}"
+}
+
+tildify() {
+    if [[ $1 = $HOME/* ]]; then
+        local replacement=\~/
+
+        echo "${1/$HOME\//$replacement}"
+    else
+        echo "$1"
+    fi
+}
+
+case $(uname -ms) in
+'Darwin x86_64')
+    target=x86_64-apple-darwin
+    ;;
+'Darwin arm64')
+    target=aarch64-apple-darwin
+    ;;
+'Linux x86_64' | *)
+    target=x86_64-unknown-linux-musl
+    ;;
```

Comment:
Shouldn't we display an error instead and quit with it is not supported? Otherwise, people might install something that will not work on their system...